File: Strings/1168.py
vezes = int(input());
count = 1

while count <= vezes:

	valores = input();
	i = 0;
	soma = 0;
	
	while i < len(valores):

		if valores[i] == "1":
			soma += 2;

		elif valores[i] == "2" or valores[i] == "3" or valores[i] == "5":
			soma += 5;

		elif valores[i] == "4":
			soma += 4;

		elif valores[i] == "6" or valores[i] == "9" or valores[i] == "0":
			soma += 6;

		elif valores[i] == "7":
			soma += 3;

		else:
			soma += 7;

		
		i += 1;
		
		
	print(str(soma) + " leds");
	count += 1;

File: Strings/1168.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;
    
    for(int x = 0; x < n; x++) {
        
        char str[101];
        cin >> str;
        int leds = 0, i = 0;
        
        while(i < 101 && str[i] != '\0') {
            if(str[i] == '2' || str[i] == '3' || str[i] == '5') {
                leds += 5;
            } else if(str[i] == '6' || str[i] == '9'|| str[i] == '0') {
                leds += 6;
            } else if(str[i] == '8') {
                leds += 7;
            } else if(str[i] == '4') {
                leds += 4;
            } else if(str[i] == '7') {
                leds += 3;
            } else if(str[i] == '1') {
                leds += 2;
            }
            i++;
        }
        cout << leds << " leds" << endl;
    }
    return 0;
}